browser/src/react-redux/App/User/User.reducer.js

```diff
@@ -0,0 +1,25 @@
+const initialState = {
+  loginError: null,
+  loggedIn: false
+}
+
+export default (state = initialState, action) => {
+  switch (action.type) {
+    case 'NEW_USER_SUBMITTED':
+      return { ...action.user }
+    case 'LOGIN_ERROR':
```

Comment:
This can be stored in React state, inside of `LoginPage.js`. No need to have this in our global store. Remember, the login success and error messages are specific to the LoginPage component. Let's get rid of this.